.cirrus.yml

```diff
@@ -0,0 +1,75 @@
+test_linux_task:
+  name: Test `patrol develop` on Linux
+  container:
+    image: ghcr.io/cirruslabs/flutter:stable
+    cpu: 6
+    memory: 12G
+    kvm: 'true'
+  env:
+    PATH: $HOME/.pub-cache/bin:${PATH}
+    EMULATOR_API_LEVEL: '34'
+    EMULATOR_ABI: google_apis_playstore;x86_64
+    EMULATOR_IMAGE: system-images;android-${EMULATOR_API_LEVEL};${EMULATOR_ABI}
+  timeout_in: 30m
+
+  setup_flutter_script: |
+    flutter precache
+    flutter --version
+  generate_gradlew_script: |
+    cd packages/patrol/example
+    flutter build apk --target lib/main.dart --debug --flavor=does-not-exist &
+    start=$SECONDS
+    until [ -e "android/gradlew" ] || [ $(($SECONDS - start)) -ge 120 ]; do sleep 2; done
+    if [ ! -e "android/gradlew" ]; then
+        echo "android/gradlew was not generated within the 2 minutes timeout"
+        exit 1
+    fi
+    kill $!
+  setup_patrol_cli_script:
+    - dart pub global activate --source path packages/patrol_cli && patrol
+  setup_emulator_script: |
+    sdkmanager --install "$EMULATOR_IMAGE"
+    sdkmanager --install emulator
+  create_emulator_script: |
+    avdmanager -s create avd -n MyAVD -k "$EMULATOR_IMAGE"
+    cat << EOF >> ~/.android/avd/MyAVD.avd/config.ini
+    hw.cpu.ncore=4
+    hw.gpu.enabled=yes
+    hw.gpu.mode=swiftshader_indirect
+    hw.ramSize=4096
+    disk.dataPartition.size=6G
+    vm.heapSize=576
+    hw.lcd.density=440
+    hw.lcd.height=2220
+    hw.lcd.width=1080
+    EOF
+  start_emulator_background_script:
+    - $ANDROID_HOME/emulator/emulator @MyAVD -verbose -no-snapshot-save -no-window -noaudio -no-boot-anim -accel on
+  wait_for_emulator_script:
+    - android-wait-for-emulator
+  patrol_develop_script: |
+    cd dev/cli_tests
+    dart pub get
+    dart patrol_develop_test.dart
+
+test_macos_task:
+  name: Test `patrol develop` on macOS
+  macos_instance:
+    image: ghcr.io/cirruslabs/macos-ventura-xcode:latest
+  env:
+    PATH: $HOME/.pub-cache/bin:${PATH}
+  timeout_in: 30m
+
+  setup_simulator_script: |
+    xcrun simctl boot "iPhone 14 Pro Max"
+  setup_flutter_script: |
+    flutter channel stable
+    flutter upgrade
+    flutter precache
+    flutter --version
```

Comment:
We'd like to enforce a specific version (version range), e.g. `3.10.x`. You can use `fvm` for this.